pkgs/development/libraries/science/math/suitesparse/default.nix

```diff
@@ -4,32 +4,37 @@
 , blas, lapack
 , metis
 , fixDarwinDylibNames
-, gnum4
+, gmp
+, mpfr
 , enableCuda ? false
 , cudatoolkit
 }:
 
+# Use compatible indexing for lapack and blas used
+assert (blas.isILP64 == lapack.isILP64);
```

Comment:
Could we move the assertion to `buildInputs` so that people can fix the situation locally using `suitesparse.override` when the assertion fails?

Reply:
Also I can't imagine a situation where someone will encounter an assertion failure and `blas` and `lapack` wouldn't be override-able.